ADRs/ADR0006-replace-badger-kv-store [WITHDRAWN].md

```diff
@@ -0,0 +1,200 @@
+# Replace Badger KV Store with PostgreSQL [WITHDRAWN]
+
```

Comment:
My only request would be to add a TOC so it's easier to get a view into all the options / sections.